R/main.R

```diff
@@ -86,8 +87,27 @@ cli_args_process_run_parameters <- function(parameters) {
         "Invalid parameters %s - all must be in form key=value",
         paste(squote(parameters[err]), collapse = ", ")))
     }
-    value <- lapply(p, function(x) parse_parameter(x[[2L]]))
-    set_names(value, vcapply(p, "[[", 1L))
+    if (command == "batch") {
```

Comment:
I think this would be cleaner as two separate functions - one for batch and one for run